digitalidentity/service.go

```diff
@@ -145,3 +152,120 @@ func GetShareQr(httpClient requests.HttpClient, sessionID string, clientSdkId, a
 
 	return share, err
 }
+
+// Get Receipt info using the supplied receiptId
+func GetReceipt(httpClient requests.HttpClient, receiptId string, clientSdkId, apiUrl string, key *rsa.PrivateKey) (share ReceiptResponse, err error) {
+	receiptUrl := requests.Base64ToBase64URL(receiptId)
+	endpoint := fmt.Sprintf(identitySessionReceiptRetrieval, receiptUrl)
+
+	headers := requests.AuthHeader(clientSdkId)
+	request, err := requests.SignedRequest{
+		Key:        key,
+		HTTPMethod: http.MethodGet,
+		BaseURL:    apiUrl,
+		Endpoint:   endpoint,
+		Headers:    headers,
+	}.Request()
+	if err != nil {
+		return share, err
+	}
+
+	response, err := requests.Execute(httpClient, request)
+	if err != nil {
+		return share, err
+	}
+	defer response.Body.Close()
+
+	responseBytes, err := io.ReadAll(response.Body)
+	if err != nil {
+		return share, err
+	}
+
+	err = json.Unmarshal(responseBytes, &share)
+
+	return share, err
+}
+
+// Get Receipt info using the supplied receiptId
```

Comment:
comment is copy/paste of the previous endpoint

Reply:
Updated